### ISSUE TITLE
Docker run panics with "Tileset 'fonts' not found"

### ISSUE THREAD

**Issue by arashgh-gh** [t-rex-tileserver/t-rex#159]
Why do I get this error on starting T-Rex image on docker? 

`thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Error(None)', src/libcore/result.rs:999:5`

Later on while using maputnik via T-Rex Viewer I get this error:

 `panicked at 'Tileset 'fonts' not found', src/libcore/option.rs:1036:5 Panic in Arbiter thread, shutting down system.  `

Shall I add somewhere fonts.json?

**Comment by pka** [comment]
Can you see the URL which is requestes?

**Comment by arashgh-gh** [comment]
I get the first error just a few seconds after starting the service in container logs and after 2 minutes or so it disappears and t-rex starts normally.

For the second error I add here more details out of the container log:

```
2019-11-13 09:44:20.099 INFO GET /maputnik.html?style=http://...:port/test.style.json HTTP/1.1 200 349 0.172894ms 10.255.0.3:49795
2019-11-13 09:44:20.354 INFO GET /maputnik-vendor.js HTTP/1.1 200 2042036 227.280808ms 10.255.0.3:49795
2019-11-13 09:44:20.395 INFO GET /maputnik.js HTTP/1.1 200 1995208 267.563979ms 10.255.0.3:49794
2019-11-13 09:44:20.734 INFO GET /img/logo-color.svg HTTP/1.1 200 2869 0.380399ms 10.255.0.3:49794
2019-11-13 09:44:20.789 INFO GET /test.style.json HTTP/1.1 200 925 0.421260ms 10.255.0.3:49795
2019-11-13 09:44:20.823 INFO GET /fonts/Roboto-Medium.ttf HTTP/1.1 200 511592 16.316123ms 10.255.0.3:49795
2019-11-13 09:44:20.828 INFO GET /fonts/Roboto-Regular.ttf HTTP/1.1 200 162876 22.184058ms 10.255.0.3:49798
thread 'actix-rt:worker:16' panicked at 'Tileset 'fonts' not found', src/libcore/option.rs:1036:5
2019-11-13 09:44:21.830 INFO GET /test.json HTTP/1.1 200 977 18.325682ms 10.255.0.3:49794
Panic in Arbiter thread, shutting down system.
thread 'actix-rt:worker:19' panicked at 'Tileset 'fonts' not found', src/libcore/option.rs:1036:5
Panic in Arbiter thread, shutting down system.
thread 'actix-rt:worker:17' panicked at 'Tileset 'fonts' not found', src/libcore/option.rs:1036:5
Panic in Arbiter thread, shutting down system.
thread 'actix-rt:worker:20' panicked at 'Tileset 'fonts' not found', src/libcore/option.rs:1036:5
Panic in Arbiter thread, shutting down system.
thread 'actix-rt:worker:7' panicked at 'Tileset 'fonts' not found', src/libcore/option.rs:1036:5
Panic in Arbiter thread, shutting down system.
```
The names of layers are listed in maputnik for styling but they are not visualized on the map in maputnik. Does it have something to do with this error?

**Comment by arashgh-gh** [comment]
Hello again,

I have encountered another problem. I followed your example for user grid and it works fine with OpenLayers (in my case in EPSG:25832). The vector tiles are cached properly if I produce them via moving in my map in OpenLayers. But if I hit the generate command for cache generation, the tiles will be visualized messy in my map. What am I doing wrong? May I open a new issue?

This is my grid:

```
[grid.user]
width = 256
height = 256
srid = 25832
units = "m"
extent = {minx = -46133.17, miny = 5048875.26857567, maxx = 1206211.10142433, maxy = 6301219.54}
resolutions = [4891.96981025128,2445.98490512564,1222.99245256282,611.496226281410,305.748113140705,152.874056570353,76.4370282851763,38.2185141425881,19.1092570712941,9.55462853564703,4.77731426782352,2.38865713391176,1.19432856695588,0.597164283477939,0.298582141738970,0.14929107086948457,0.0746455354347426,0.0373227677173713]
origin = "TopLeft"
```
Many thanks.

**Comment by pka** [comment]
For the first error: Could you run Docker with `docker run -e "RUST_BACKTRACE=1"`... to get more information?

**Comment by arashgh-gh** [comment]
I have no explanation but we made the container with the same conditions again and it runs without the first error. It also starts much faster than before.

**Comment by mosinuga** [comment]
Were you able to resolve the second error?

**Comment by arashgh-gh** [comment]
> Were you able to resolve the second error?
No, I have still the second one.

**Comment by pka** [comment]
Sorry, currently not enough time for analyzing this problem further.

**Comment by elvios** [comment]
I get the same error, when opening Maputnik. The error comes when the browser tries to get `/fonts.json`. When I just try `/fonts.json` in the browser, then this error comes:

```
thread 'actix-rt:worker:38' panicked at 'Tileset 'fonts' not found', src/libcore/option.rs:1036:5
```

Also, when i run the docker-container with `RUST_BACKTRACE=1`, this is the output (10 times, I guess because I have 10 threads?):

```
thread 'actix-rt:worker:38' panicked at 'Tileset 'fonts' not found', src/libcore/option.rs:1036:5
stack backtrace:
   0: <unknown>
   1: <unknown>
   2: <unknown>
   3: <unknown>
   4: <unknown>
   5: <unknown>
   6: <unknown>
   7: <unknown>
   8: <unknown>
   9: <unknown>
  10: <unknown>
  11: <unknown>
  12: <unknown>
  13: <unknown>
  14: <unknown>
  15: <unknown>
  16: <unknown>
  17: <unknown>
  18: <unknown>
  19: <unknown>
  20: <unknown>
  21: <unknown>
  22: <unknown>
  23: <unknown>
  24: <unknown>
  25: <unknown>
  26: <unknown>
  27: <unknown>
  28: start_thread
  29: clone
Panic in Arbiter thread, shutting down system.
```

**Comment by pka** [comment]
Fixed in 736a0a0
